tests/test_search_api_client.py

```diff
@@ -77,6 +77,63 @@ def service():
     }
 
 
+@pytest.fixture
+def brief():
```

Comment:
Do these tests actually run against a Search API instance? If not, I don't really get how this fixture actually tests anything? (I get we're copying the G-Cloud fixture here; I don't understand that either.)

Reply:
I think it's just to make the test as realistic as possible. In the tests, we're calling the method on the search_api_client, and passing in the variables it normally takes. In the tests that use this fixture, nothing is actually being done with it (it just goes into the data json), but it feels like we should test it with something as close to real as possible. In the future if someone changed the index method to do something different, it'd be good to have "real" brief in the test to catch any issues that might arise. Or something.

Reply:
Hmm, yeah I tend to think this is just yet another fixture that goes out of date as we iterate frameworks, in which case it might as well contain as little as possible, even if that makes it unrealistic. ISTM that the API _client_ should _never_ do anything much with the data it passes through (any important logic should be in the service)...

If we change this we should probably rip out the data from services fixture too. @risicle what you think?

Reply:
Tend to agree unless we can think of any gotchas where we *are* dependent on some brief properties...?

Reply:
Yep, that makes sense. I'll strip it out and look at taking the service out too.